internal/testutil/acc/tpf_converter_test.go

```diff
@@ -0,0 +1,143 @@
+package acc_test
+
+import (
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestConvertAdvancedClusterToTPF(t *testing.T) {
+	var (
+		input = `
+			resource "mongodbatlas_advanced_cluster" "cluster2" {
+				project_id   = "MY-PROJECT-ID"
```

Comment:
Are we covering all blocks?
Maybe missing `bi_connector_config`, `tags`, `labels`, etc.?